openIt/ngsild-payloads/key.jsonld

```diff
@@ -21,8 +21,14 @@
     "type": "Property",
     "value": "P1D"
   },
-  "category": {
-    "type": "Property",
-    "value": "Fire Department"
-  }
+  "isAvailableForOrganization": [
```

Comment:
if it is a multi-instance attribute, you have to use `datasetId` to distinguish each instance

Reply:
Understood